README.md

```diff
@@ -6,6 +6,8 @@
 [![PyPi](https://img.shields.io/pypi/v/pylance)](https://pypi.org/project/pylance/)
 ![Python versions](https://img.shields.io/pypi/pyversions/pylance)
 
+[![](https://dcbadge.vercel.app/api/server/qs7kJxmMjU?style=flat)](https://discord.gg/qs7kJxmMjU)
```

Comment:
Change link to: https://discord.gg/zMM32dvNtd

Reply:
Updated.